local/compose/create.go

```diff
@@ -44,6 +44,20 @@ func (s *composeService) Create(ctx context.Context, project *types.Project) err
 		return err
 	}
 
+	if err := s.ensureProjectNetworks(ctx, project); err != nil {
+		return err
+	}
+
+	if err := s.ensureProjectNetworks(ctx, project); err != nil {
+		return err
+	}
```

Comment:
Ran twice, we can be sure networks will be set :P

Reply:
Fixed, the call to ensure volumes had been replaced by duplicated call to network at some point...